lemur/certificates/views.py

```diff
@@ -1398,7 +1398,7 @@ def __init__(self):
         self.reqparse = reqparse.RequestParser()
         super(CertificateRevoke, self).__init__()
 
-    @validate_schema(None, None)
+    @validate_schema(certificate_revoke_schema, None)
     def put(self, certificate_id, data=None):
         """
         .. http:put:: /certificates/1/revoke
```

Comment:
should we update the expected input, in terms of `crl_reason`?

Reply:
Added [here](https://github.com/Netflix/lemur/pull/3277/files#diff-88f1b5db302abda0bddba15f5af7d9c1aed03efcdd87d603bd0c3a96c41d772fR1417)